docs/architecture.md

```
@@ -113,35 +157,44 @@ awa_model::insert() / insert_with() / insert_many()
-Insert accepts a `PgExecutor`, so it works inside an existing transaction — the job becomes visible only when the outer transaction commits. This is the transactional enqueue pattern.
+Insert accepts a `PgExecutor`, so it works inside an existing transaction — the
+job becomes visible only when the outer transaction commits. This is the
+transactional enqueue pattern.
 
 ### Batch Insert via COPY
 
-For high-throughput ingestion (10K+ jobs), `insert_many_copy` uses PostgreSQL's COPY protocol via a staging table approach (see ADR-008):
+For high-throughput ingestion (10K+ jobs), `insert_many_copy` uses PostgreSQL's
+COPY protocol via a staging table approach (see ADR-008):
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)